packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -63,6 +63,7 @@ export const excluded_notifications = ['contract_sold', 'has_changed_two_fa'];
 export const maintenance_notifications = ['system_maintenance', 'site_maintenance'];
 
 export const priority_toast_messages = [
+    'aud_account_closure',
```

Comment:
should we mention aud here?

Reply:
@utkarsha-deriv, @amina-deriv, @matin-deriv 
I posted the message regarding current BE implementation and the goal of the task. will refactor the code to remove the mentioning of AUD once we agree with the requirements